composer.json

```
@@ -4,7 +4,7 @@
     "license" : "BSD-2-Clause",
     "homepage": "https://github.com/Bacon/BaconQrCode",
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
         "ext-iconv": "*",
         "dasprid/enum": "^1.0"
```

```suggestion
        "dasprid/enum": "^1.0.3"
```
To be sure the package is compatible

---

I’m not sure if it is the responsibility of this project to ensure that it’s dependency is compatible with all supported platform versions. I think I would keep it as `^1.0`, but I will change it to whatever @DASPRiD prefers.

---

I don't have a strong preference.

But since the enum 1.0.3 library supports both PHP versions, it could make sense to make that a required version, especially since it's just a patch version. So even if another dependency is using dasprid/enum, it would not be BC-breaking for a parent project.